pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java

```diff
@@ -70,8 +70,8 @@ private static void postBuildCheck(String stagingPath, String productName) {
                     + CollectionUtils.subtract(newDependencies, oldDependencies);
             String oldBuildInfo = "Community Dependencies present in old build which are not present in new build are "
                     + CollectionUtils.subtract(oldDependencies, newDependencies);
-            log.info("Build info for new build is {}", newBuildInfo);
-            log.info("Build info for old build is {}", oldBuildInfo);
+            log.info("Build info for new build is " + newBuildInfo);
+            log.info("Build info for old build is " + oldBuildInfo);
```

Comment:
Still got logging changes.... ?

Reply:
I will raise a new PR with squashed commits and formatting changes. Running mvn clean install didn't work too :( Need to find some other way I think